src/radical/pilot/configs/resource_ncsa.json

```diff
@@ -0,0 +1,176 @@
+{
+    "delta":
+    {
+        "description"                 : "132 standard CPU (AMD  EPYC 7763) nodes",
+        "schemas"                     : ["local", "batch", "interactive"],
+        "local"                       :
+        {
+            "job_manager_endpoint"    : "slurm://delta.ncsa.illinois.edu/",
+            "filesystem_endpoint"     : "file://delta.ncsa.illinois.edu/"
+        },
+        "batch"                       :
+        {
+            "job_manager_endpoint"    : "fork://localhost/",
+            "filesystem_endpoint"     : "file://localhost/"
+        },
+        "interactive"                 : "batch",
+        "default_queue"               : "cpu",
+        "cores_per_node"              : 128,
+        "resource_manager"            : "SLURM",
+        "agent_scheduler"             : "CONTINUOUS",
+        "agent_spawner"               : "POPEN",
+        "launch_methods"              : {
+                                         "order": ["SRUN"],
+                                         "SRUN" : {}
+                                        },
+        "pre_bootstrap_0"             : [
+                                         "module reset",
+                                         "module load python"
+                                        ],
+        "default_remote_workdir"      : "/scratch/%(pd.project)s/$USER",
+        "python_dist"                 : "default",
+        "virtenv_mode"                : "create",
```

Comment:
would it make sense to default to the `local` `virtenv_mode`?

Reply:
yeah, will do

Reply:
done